addon/components/masked-input.js

```diff
@@ -17,84 +18,88 @@ function isRedo(e) {
 
 const PROTECTED_ATTRS = ['change', 'keyDown', 'keyPress', 'paste'];
 
-export default Ember.TextField.extend({
+export default TextField.extend({
   mask: null,
   formatCharacters: null,
   placeholderChar: '_',
 
-  'on-change': Ember.K,
+  'on-change'() {
+    return this;
+  },
 
-  validateAttrs: Ember.on('init', function() {
+  init() {
+    this._super(...arguments);
     PROTECTED_ATTRS.forEach((key) => {
       if (this.attrs[key]) {
         if (console && console.warn) {
           console.warn(`WARNING: Attribute "${key}" of {{masked-input}} helper should not be overriden${key === 'change' ? ', please use "on-${key}" for custom event handlers' : ''}`);
         }
       }
     });
-  }),
+  },
 
-  applyMaskToInitialValue: Ember.on('didInsertElement', function() {
-    const el = this.get('element');
+  didInsertElement() {
+    this._super(...arguments);
+    let el = this.get('element');
     el.value = this._getDisplayValue();
-  }),
+  },
 
-  _inputMask: Ember.computed('mask', 'value', 'formatCharacters', 'placeholderChar', function() {
+  _inputMask: computed('mask', 'value', 'formatCharacters', 'placeholderChar', function() {
     let options = {
       pattern: this.get('mask'),
       value: this.get('value'),
       formatCharacters: this.get('formatCharacters')
     };
-    const placeholderChar = this.get('placeholderChar');
+    let placeholderChar = this.get('placeholderChar');
```

Comment:
Hi there! Thanks for your PR, I'll be glad to merge this. One question though: why replace `const` with `let` throughout the code?